src/scripts/items/PokemonItem.ts

```diff
@@ -66,7 +66,7 @@ class PokemonItem extends CaughtIndicatingItem {
     }
 
     get displayName(): string {
-        return this._translatedName();
+        return this._displayName ?? this._translatedName();
```

Comment:
We should probably still be showing the `_translatedName` here, and default that to using `_displayName ?? pokemon`

Reply:
fixed